README.md

```diff
@@ -7,15 +7,32 @@ This repository contains the implementation of the Concordium browser wallet, wh
 -   [browser-wallet-api-helpers](./packages/browser-wallet-api-helpers) is types and helper methods associated with the browser wallet api, which can be installed in projects that want to utilize the browser wallet API. This is published as a package on NPM as [@concordium/browser-wallet-api-helpers](https://www.npmjs.com/package/@concordium/browser-wallet-api-helpers).
 -   [browser-wallet-message-hub](./packages/browser-wallet-message-hub) is utilities used for internal communication between the separate contexts in the chrome extension.
 
+-   [examples](./examples), a collection of example dapps that can be used with the browser wallet.
+
 ## Installing
 
+### Prerequisites
+
+-   Node.js version 18.14.2
+-   Typescript installed globally `npm -g install typescript` (Used for running the post install script of wallet-common-helpers during `yarn install`)
+-   Yarn installed globally `npm -g install yarn` (The repository embeds yarn 3.2.0, but needs a global version for easy usage)
+
+### Setup
+
 -   Clone repository
 -   Install dependencies by running `yarn install`
+-   Build concordium api helpers by running `yarn build:api-helpers` (The wallet and examples require this package to be built, before they can be built)
```

Comment:
Suggestion: If we add one additional line then people can get the browser wallet built for e.g. testing/reviewing without going through the second readMe.
```yarn build:(dev|prod)```